src/IscalculatingRight.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const IscalculatingRight = (userInput) => {
```

Comment:
Consider renaming the function 'IscalculatingRight' to something more descriptive and camelCase like 'validateUserInput'.